_posts/2023-12-07-Latin-American-students-drive-innovation-with-open-data.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Hackathon Co-Afina 2023 - Latin American Students Drive Innovation with Open Data
+authors:
+- camachoreina
```

Comment:
I think Alexander Martinez is mxrtinez on the website

Reply:
Found and added him. Thanks!